atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -64,6 +64,7 @@
 class AddLabelsPlugin(PreBuildPlugin):
     key = "add_labels_in_dockerfile"
     is_allowed_to_fail = False
+    explicit_labels = ['release', 'version']
```

Comment:
Trying to think of the best name for this variable. What about `overwrite_whitelist`?